### PR TITLE
SymPy 1.1 release

### PR DESCRIPTION
**Pull request by asmeurer**
Do not merge this. I will merge it when the release is done.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -284,3 +284,9 @@ Jainul Vaghasia <jainulvaghasia@gmail.com> Jainul Vaghasia <jainulvaghasia@D-10-
 Arif Ahmed <arif.ahmed.5.10.1995@gmail.com> ArifAhmed1995 <arif.ahmed.5.10.1995@gmail.com>
 Pranjal Tale <pranjaltale16@gmail.com> = <pranjaltale16@gmail.com>
 jtoka <jason.tokayer@gmail.com> jtoka <jason.tokayer@capitalone.com>
+Theodore Han <theodorehan@hotmail.com> LAPTOP-4VOQQ6GI\han <theodorehan373@gmail.com>
+Theodore Han <theodorehan@hotmail.com> Theodore Han <tqhan317@gmail.com>
```
Comment (moorepants) [inline .mailmap]: A repeat.

Comment (moorepants) [inline .mailmap]: Oh, I forgot the purpose of this. Repeat is fine.

Comment (asmeurer) [inline .mailmap]: This is .mailmap. Every unique name/email in the git history has to have a line here.

**Comment by smichr** [comment]
I would really like to see #12881 and #12740 make it into this. They both contain bug fixes. I need #12740 to make piecewise integration work -- there are about a dozen outstanding issues related to that -- and #12881 is a really easy fix to some issues related to Poly free symbols and solving biquadratic system of equations. #12740 is running tests now after a merge.

**Comment by asmeurer** [comment]
I'm sorry, but I'm going to make a call that it's too late for this. I had been planning on making the final release either today or tomorrow, and those changes are rather much to merge post-release candidate. Neither PR seems release critical (correct me if I am wrong). Actually, for the one PR (the piecewise one), I would prefer to see it sit in master for a little while before releasing it, as it seems like a significant change that could use some stabilizing.

**Comment by smichr** [comment]
> PR seems release critical (correct me if I am wrong).

It all depends what you think is critical. #12881 is a +47/-21 line change that corrects the ltrim function so it doesn't trim non-dummies (which it does now), corrects `solve_biquadratic` so it actually gives a solution without errors for cases involving infinite solutions, and gives the correct free symbols for a Poly -- it no longer reports dummy symbols that don't appear in the expression. I think those changes are very solid. I'm not going to cry if they don't go in...somebody else might, however, when they get bit by those issues.

I agree that the other PR can sit in master for a while.

**Comment by asmeurer** [comment]
This is ready for merging into master.

**Comment by moorepants** [comment]
Nice work on this! Congrats.

**Pull request merged**